generator/vulkan/render.zig

```diff
@@ -910,17 +962,44 @@ fn Renderer(comptime WriterType: type) type {
         }
 
         fn renderWrappers(self: *Self) !void {
-            try self.renderWrappersOfDispatchType("BaseWrapper", .base);
-            try self.renderWrappersOfDispatchType("InstanceWrapper", .instance);
-            try self.renderWrappersOfDispatchType("DeviceWrapper", .device);
+            try self.renderWrappersOfDispatchType(.base);
+            try self.renderWrappersOfDispatchType(.instance);
+            try self.renderWrappersOfDispatchType(.device);
         }
 
-        fn renderWrappersOfDispatchType(self: *Self, name: []const u8, dispatch_type: CommandDispatchType) !void {
+        fn renderWrappersOfDispatchType(self: *Self, dispatch_type: CommandDispatchType) !void {
+            const name = switch (dispatch_type) {
+                .base => "Base",
+                .instance => "Instance",
+                .device => "Device",
+            };
+
             try self.writer.print(
-                \\pub fn {s}(comptime Self: type) type {{
+                \\pub fn {s}Wrapper(comptime cmds: anytype) type {{
+                \\    comptime var fields: [cmds.len]std.builtin.TypeInfo.StructField = undefined;
+                \\    inline for (cmds) |cmd, i| {{
+                \\        const cmd_camel_case = snakeToCamel{s}(cmd);
```

Comment:
I figured I'd create a custom `snakeToCamel{}` function for each type, as it's easier/faster than converting arbitrary strings, and actually is the only thing here I think that guarantees type safety, as it requires an input of the specified type.

Does make the code footprint larger, though.